### ISSUE TITLE
metaSPADES 3.12 Hybrid Assembly hanging at hybrid aligner

### ISSUE THREAD

**Issue by alexcritschristoph** [ablab/spades#137]
I've been running a pacbio-illumina hybrid assembly for a complex metagenome with the latest metaspades 3.12. This is a very complex dataset, and includes 800k pacbio reads totalling ~6.1 GB. The Illumina assembly steps finish within 5 days, but spades has been hanging at the hybrid aligner stage for days now: (It ran for 3 days - I then stopped the run and re-ran it with `--continue` and it's been hanging there again for another day)
 
```
 11:16:07.364    86G / 242G  INFO   StageManager             (stage.cpp                 : 132)   STAGE == Hybrid Aligning
 11:16:07.364    86G / 242G  INFO   HybridAligning           (hybrid_aligning.cpp       : 322)   Hybrid library detected: #1
 11:16:07.364    86G / 242G  INFO    General                 (hybrid_aligning.cpp       : 288)   Aligning long reads with bwa-mem based aligner
 12:35:13.048    92G / 242G  INFO    General                 (hybrid_aligning.cpp       : 268)   Prepared batch 0 of 50000 reads.
```
I'm worried that if the first 50k batch takes so long, it will take an infeasibly long time to align all 800k reads. Also noticing that during this stage spades is maxing out at 15 threads, while I pass it 60. Full command:

```
metaspades.py --pe1-1 PE.1.fastq.gz --pe1-2 PE.2.fastq.gz -m 1000 -t 60 -k 21,33,55,77,99,127 --pacbio pacbio.fasta -o metaspades312_pacbio
```

I hope it's not just the sheer complexity of the graph that the reads are being aligned to that takes so long and that there's something that can be done to speed up the hybrid step here.

**Comment by asl** [comment]
Will you please attach the full spades.log file?

**Comment by alexcritschristoph** [comment]
Here it is! You'll notice that I reset and ran ``--continue`` to try and figure out if the running process was simply dead a day or so ago
[spades.log](https://github.com/ablab/spades/files/2174175/spades.log)

**Comment by asl** [comment]
This is very weird. Will it be possible to share your data with us? So we could reproduce, investigate and probably fix the issue?

**Comment by alexcritschristoph** [comment]
Hi Anton,
Yes, for sure - just working on getting it uploaded to a file server for sharing. I'll send you an email with a link.
Thanks!
Alex

**Comment by asl** [comment]
Hi Alex,

I believe we already identified the problem and the fix will be in SPAdes 3.12.1

**Comment by asl** [comment]
Fixed in 3.13.0
